Server crashed during inserting

**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
```
[2021/05/21 03:19:28.339 +00:00] [DEBUG] [timestamp.go:105] ["TimestampAllocator pickCanDoFunc"] [need=1] [total=65531] [remainReqCnt=0]
[2021/05/21 03:19:28.339 +00:00] [DEBUG] [id.go:143] ["IDAllocator pickCanDoFunc"] [need=1] [total=199994] [remainReqCnt=0]
[2021/05/21 03:19:28.339 +00:00] [DEBUG] [impl.go:1076] [Insert] [role=ProxyNode-1] [msgID=425084099483077958] [timestamp=0] [db=] [collection=local_1m_128_l2] [partition=_default] [len(RowData)=50000] [len(RowIDs)=0]
[2021/05/21 03:19:28.339 +00:00] [DEBUG] [id.go:143] ["IDAllocator pickCanDoFunc"] [need=50000] [total=199993] [remainReqCnt=0]
[2021/05/21 03:19:28.341 +00:00] [DEBUG] [segment_allocator.go:190] ["dataservice::alloc: "] [segMeta.NumRows=24914] [totalOfAllocations=0]
[2021/05/21 03:19:28.341 +00:00] [DEBUG] [repack_func.go:125] [proxynode] ["repackFunc, reqSegCountMap, reqID"=425084099483077958] [mapinfo="{\"425084101278564353\":24917}"]
[2021/05/21 03:19:28.341 +00:00] [DEBUG] [segment_allocator.go:190] ["dataservice::alloc: "] [segMeta.NumRows=25086] [totalOfAllocations=0]
[2021/05/21 03:19:28.341 +00:00] [DEBUG] [repack_func.go:125] [proxynode] ["repackFunc, reqSegCountMap, reqID"=425084099483077958] [mapinfo="{\"425084101278564354\":25083}"]
[2021/05/21 03:19:28.341 +00:00] [DEBUG] [repack_func.go:189] [proxynode] ["threshold of message size: "=524288]
[2021/05/21 03:19:28.420 +00:00] [DEBUG] [impl.go:1086] ["Insert Done"] [] [role=ProxyNode-1] [msgID=425084099483077958] [timestamp=0] [db=] [collection=local_1m_128_l2] [partition=_default] [len(RowData)=50000] [len(RowIDs)=50000]
[2021/05/21 03:19:28.436 +00:00] [DEBUG] [timestamp.go:105] ["TimestampAllocator pickCanDoFunc"] [need=1] [total=65530] [remainReqCnt=0]
[2021/05/21 03:19:28.437 +00:00] [DEBUG] [id.go:143] ["IDAllocator pickCanDoFunc"] [need=1] [total=199992] [remainReqCnt=0]
[2021/05/21 03:19:28.437 +00:00] [DEBUG] [impl.go:380] [GetCollectionStatistics] [role=ProxyNode-1] [msgID=425084099483077960] [timestamp=425084103546175494] [db=] [collection=local_1m_128_l2]
[2021/05/21 03:19:28.438 +00:00] [DEBUG] [impl.go:387] ["GetCollectionStatistics Done"] [] [role=ProxyNode-1] [msgID=425084099483077960] [timestamp=425084103546175494] [db=] [collection=local_1m_128_l2]
[2021/05/21 03:19:28.486 +00:00] [DEBUG] [timestamp.go:105] ["TimestampAllocator pickCanDoFunc"] [need=0] [total=65529] [remainReqCnt=0]
[2021/05/21 03:19:28.487 +00:00] [DEBUG] [timestamp.go:105] ["TimestampAllocator pickCanDoFunc"] [need=0] [total=65536] [remainReqCnt=0]
[2021/05/21 03:19:28.487 +00:00] [DEBUG] [timestamp.go:105] ["TimestampAllocator pickCanDoFunc"] [need=1] [total=65536] [remainReqCnt=0]
[2021/05/21 03:19:28.488 +00:00] [DEBUG] [timetick.go:68] [proxyservice] ["msg type"=TimeTick]
[2021/05/21 03:19:28.490 +00:00] [DEBUG] [collection_replica.go:236] ["updating segment"] ["Segment ID"=425084101278564353] [numRows=24917]
[2021/05/21 03:19:28.490 +00:00] [DEBUG] [collection_replica.go:236] ["updating segment"] ["Segment ID"=425084101278564354] [numRows=25083]
[2021/05/21 03:19:28.490 +00:00] [DEBUG] [flow_graph_insert_buffer_node.go:749] ["Updating segments statistics..."]
[2021/05/21 03:19:28.491 +00:00] [DEBUG] [stats_handler.go:47] ["stats_handler update segment"] [segmentID=425084101278564353] [State=Growing]
[2021/05/21 03:19:28.492 +00:00] [DEBUG] [stats_handler.go:47] ["stats_handler update segment"] [segmentID=425084101278564354] [State=Growing]
[2021/05/21 03:19:28.503 +00:00] [ERROR] [server.go:357] ["Fail to store current success pos for Stats stream"] [pos="channel_name:\"dataservice-statistics-channel\" msgID:\"k\\001\\\\\\033\\2743\\346\\005\" "] [error="invalid stream type"] [stack="github.com/milvus-io/milvus/internal/log.Error\n\t/go/src/github.com/milvus-io/milvus/internal/log/global.go:42\ngithub.com/milvus-io/milvus/internal/dataservice.(*Server).startStatsChannel\n\t/go/src/github.com/milvus-io/milvus/internal/dataservice/server.go:357"]
[2021/05/21 03:19:28.687 +00:00] [DEBUG] [timestamp.go:105] ["TimestampAllocator pickCanDoFunc"] [need=1] [total=65535] [remainReqCnt=0]
[2021/05/21 03:19:28.887 +00:00] [DEBUG] [timestamp.go:105] ["TimestampAllocator pickCanDoFunc"] [need=1] [total=65534] [remainReqCnt=0]
[2021/05/21 03:19:28.987 +00:00] [DEBUG] [flow_graph_insert_buffer_node.go:473] ["---insert buffer status---"]
[2021/05/21 03:19:28.987 +00:00] [DEBUG] [flow_graph_insert_buffer_node.go:480] ["seg buffer status"] [segmentID=425084101278564354] ["buffer size"=50169]
[2021/05/21 03:19:28.987 +00:00] [DEBUG] [flow_graph_insert_buffer_node.go:480] ["seg buffer status"] [segmentID=425084101278564353] ["buffer size"=49831]
[2021/05/21 03:19:28.987 +00:00] [DEBUG] [flow_graph_insert_buffer_node.go:537] [". Insert Buffer full, auto flushing "] ["num of rows"=49831]
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1325c80]

goroutine 681 [running]:
github.com/milvus-io/milvus/internal/datanode.(*insertBufferNode).getCollMetabySegID(0xc00094ba40, 0x5e633bce8600001, 0xc008e00100, 0x1, 0x1)
	/go/src/github.com/milvus-io/milvus/internal/datanode/flow_graph_insert_buffer_node.go:796 +0x60
github.com/milvus-io/milvus/internal/datanode.(*insertBufferNode).Operate(0xc00094ba40, 0xc008492d10, 0x1, 0x1, 0x23a33e0, 0xc00178e860, 0x0)
	/go/src/github.com/milvus-io/milvus/internal/datanode/flow_graph_insert_buffer_node.go:540 +0x5e0a
github.com/milvus-io/milvus/internal/util/flowgraph.(*nodeCtx).Start(0xc0002fd9d0, 0x23ccf60, 0xc0005ad740, 0xc000638f70)
	/go/src/github.com/milvus-io/milvus/internal/util/flowgraph/node.go:73 +0x137
created by github.com/milvus-io/milvus/internal/util/flowgraph.(*TimeTickedFlowGraph).Start
	/go/src/github.com/milvus-io/milvus/internal/util/flowgraph/flow_graph.go:75 +0xfe

```

**Steps/Code to reproduce behavior**
1. create collection
2. insert 50,000 entities per request
3. after 100,000 entities inserted, server crashed

client log:

```
2021-05-21 11:18:18,388 - milvus_benchmark.main:83 - DEBUG - Get runner
2021-05-21 11:18:29,437 - milvus_benchmark.main:87 - INFO - Prepare to run cases
2021-05-21 11:18:29,462 - milvus_benchmark.client:128 - INFO - Create collection: <local_1m_128_l2> successfully
2021-05-21 11:18:29,467 - milvus_benchmark.main:89 - INFO - Start run case
2021-05-21 11:18:30,023 - milvus_benchmark.runners.base:74 - DEBUG - Start id: 0, end id: 50000
2021-05-21 11:18:36,077 - milvus_benchmark.client:49 - DEBUG - Milvus insert run in 6.05s
2021-05-21 11:18:36,087 - milvus_benchmark.client:364 - DEBUG - Row count: 0 in collection: <local_1m_128_l2>
2021-05-21 11:18:36,092 - milvus_benchmark.runners.base:87 - DEBUG - 0
2021-05-21 11:18:36,656 - milvus_benchmark.runners.base:74 - DEBUG - Start id: 50000, end id: 100000
2021-05-21 11:18:42,620 - milvus_benchmark.client:49 - DEBUG - Milvus insert run in 5.96s
2021-05-21 11:18:42,630 - milvus_benchmark.client:364 - DEBUG - Row count: 50000 in collection: <local_1m_128_l2>
2021-05-21 11:18:42,635 - milvus_benchmark.runners.base:87 - DEBUG - 50000
2021-05-21 11:18:43,187 - milvus_benchmark.runners.base:74 - DEBUG - Start id: 100000, end id: 150000

Addr [192.168.1.239:19530] bulk_insert
RPC error: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "Socket closed"
	debug_error_string = "{"created":"@1621567124.713228022","description":"Error received from peer ipv4:192.168.1.239:19530","file":"src/core/lib/surface/call.cc","file_line":1068,"grpc_message":"Socket closed","grpc_status":14}"
>
	{'API start': '2021-05-21 11:18:43.204736', 'RPC start': '2021-05-21 11:18:43.204750', 'RPC error': '2021-05-21 11:18:44.715265'}
2021-05-21 11:18:44,715 - milvus.client.grpc_handler:71 - ERROR - 
Addr [192.168.1.239:19530] bulk_insert
RPC error: <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "Socket closed"
	debug_error_string = "{"created":"@1621567124.713228022","description":"Error received from peer ipv4:192.168.1.239:19530","file":"src/core/lib/surface/call.cc","file_line":1068,"grpc_message":"Socket closed","grpc_status":14}"
>
	{'API start': '2021-05-21 11:18:43.204736', 'RPC start': '2021-05-21 11:18:43.204750', 'RPC error': '2021-05-21 11:18:44.715265'}
2021-05-21 11:18:44,721 - milvus_benchmark.client:145 - ERROR - <_MultiThreadedRendezvous of RPC that terminated with:
	status = StatusCode.UNAVAILABLE
	details = "Socket closed"
	debug_error_string = "{"created":"@1621567124.713228022","description":"Error received from peer ipv4:192.168.1.239:19530","file":"src/core/lib/surface/call.cc","file_line":1068,"grpc_message":"Socket closed","grpc_status":14}"
>

```
**Expected behavior**
A clear and concise description of what you expected to happen.

**Method of installation**
- [ ] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Software conditions (OS, CPU, GPU, Memory)


- Milvus version (master or released version)


**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

--- a/internal/datanode/flow_graph_insert_buffer_node.go
+++ b/internal/datanode/flow_graph_insert_buffer_node.go
@@ -793,6 +793,7 @@ func (ibNode *insertBufferNode) getCollMetabySegID(segmentID UniqueID) (meta *et
 	if err != nil {
 		return
 	}
+	meta = &etcdpb.CollectionMeta{}
 	meta.ID = ret.collectionID
 
 	coll, err := ibNode.replica.getCollectionByID(ret.collectionID)

